src/test/test_utils.hpp

```diff
@@ -380,6 +380,44 @@ void prepareKFSInferInputTensor(::KFSRequest& request, const std::string& name,
         }
     }
 }
+
+template <>
+inline void prepareKFSInferInputTensor<bool>(::KFSRequest& request, const std::string& name, const std::tuple<ovms::signed_shape_t, const std::string>& inputInfo,
+    const std::vector<bool>& data, bool putBufferInInputTensorContent) {
```

Comment:
This puts bool values in input content regardless of `putBufferInInputTensorContent`. Is it intended and if so why?

Reply:
Throw in case this is false?